src/createClient.ts

```diff
@@ -1183,6 +1241,28 @@ export class Client<TDocuments extends PrismicDocument = PrismicDocument> {
 			url.searchParams.set("access_token", this.accessToken);
 		}
 
+		if (
+			params?.optimize?.repositoryRequests ??
+			this.optimize.repositoryRequests
+		) {
+			// Add a URL parameter to fingerprint the URL. The URL
+			// should be valid for the duration of
+			// REPOSITORY_CACHE_TTL.
+			//
+			// This fingerprint works as a second "cache" that
+			// works outside `getRepositoryCache()`. This is useful
+			// when multiple client instances exist with an
+			// external cache; all clients can share the same URL
+			// cache key for the duration of REPOSITORY_CACHE_TTL.
+			url.searchParams.set(
+				OPTIMIZE_REPOSITORY_REQUESTS_VALID_UNTIL_PARAMETER_NAME,
+				(
+					Math.ceil(Math.floor(Date.now() / REPOSITORY_CACHE_TTL)) *
```

Comment:
I think I'm missing something, but why do we need to ceil something that was floored just before?

Reply:
I don't see where we add the `RESPOSITORY_CACHE_TTL` to the 5s-rounded TS here 🤔

This, `Date.now() >= Math.ceil(Math.floor(Date.now() / REPOSITORY_CACHE_TTL)) * REPOSITORY_CACHE_TTL`, seems to be always true

Reply:
Haha that's for catching that. The formula was different before and required `ceil` + `floor`, but it's no longer required after some simplification.

Your `+ REPOSITORY_CACHE_TTL` is correct; I must have copied it wrong from my sandbox.